__tests__/app.test.js

```diff
@@ -50,4 +50,52 @@ describe('GET /api/categories', () => {
             expect(msg).toBe('End-point Not Found');
         });
     });
-});
\ No newline at end of file
+});
+
+describe('GET /api/reviews/:review_id', () => {
+    test('200: Ensures that what is passed IS an Object', () =>{
+        return request(app)
+        .get('/api/reviews/2')
+        .expect(200)
+        .then(({body}) => {
+            const reviews = body.reviews;
+            expect(reviews).toBeInstanceOf(Object);
+        });
+    });
+    test('200: Accepts the review with the given id', () => {
+        return request(app)
+        .get('/api/reviews/2')
+        .then(({body}) => {
+            const {review}= body.reviews;
```

Comment:
This test and destructuring need to be tweaked. I would console.log your review here and just make sure it is what you think it is.

Also, expect.objectContaining({}) does not automatically assert against an object - it just creates one. We would still need to use .toEqual here in order to compare them. Alternatively, you may want to use .toMatchObject, which does the comparison implicitly. Let me know if this doesn't make sense, though!